script/release_utils.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+
+function commit_list {
+  local tag=${1:?}
+  local repo_domain=${2:?}
+  local repo_name=${3:?}
+  git fetch --tags
+  local previous_tag=`curl -H "Authorization: token $ACCESS_TOKEN" -s https://api.github.com/repos/$repo_domain/$repo_name/tags | jq --raw-output '.[1].name'`
```

Comment:
`git describe` doesn't work because the repository has been "shallowed" (it doesn't contain the tags locally):

```
circleci@94e76f7b3045:~/project$ git describe --abbrev=0 --tags
fatal: No names found, cannot describe anything.
```

so I need to use the API

Reply:
Thanks @andresmgot 

I see, but we do `git fetch --tags` shouldn't we able to get the tag by for example doing `git describe --tags $(git rev-list --tags --max-count=1)`

```
git clone --depth 1 git@github.com:kubeapps/kubeapps.git
git fetch --tags
git describe --tags $(git rev-list --tags --max-count=1)
v1.0.0-alpha.4
```

Reply:
You are right. I was trying with my repository that doesn't have tags, in the `kubeapps` one work as expected.